cfgov/ask_cfpb/models/django.py

```diff
@@ -183,6 +183,59 @@ class Meta:
         verbose_name_plural = 'Categories'
 
 
+class SubCategory(models.Model):
+    name = models.CharField(max_length=255)
+    name_es = models.CharField(max_length=255, null=True, blank=True)
+    slug = models.SlugField()
+    slug_es = models.SlugField(
+        null=True,
+        blank=True,
+        help_text="This field is not currently used on the front end.")
+    weight = models.IntegerField(default=1)
+    description = RichTextField(
+        blank=True,
+        help_text="This field is not currently displayed on the front end.")
+    description_es = RichTextField(
+        blank=True,
+        help_text="This field is not currently displayed on the front end.")
+    more_info = models.TextField(
+        blank=True,
+        help_text="This field is not currently displayed on the front end.")
+    parent = models.ForeignKey(
+        Category,
+        null=True,
+        blank=True,
+        default=None,
+        related_name='subcategories')
+    related_subcategories = models.ManyToManyField(
+        'self',
+        blank=True,
+        default=None,
+        help_text="Maximum 3 related subcategories"
+    )
+
+    panels = [
+        FieldPanel('name', classname="title"),
+        FieldPanel('slug'),
+        FieldPanel('description'),
+        FieldPanel('name_es', classname="title"),
+        FieldPanel('slug_es'),
+        FieldPanel('description_es'),
+        FieldPanel('weight'),
+        FieldPanel('more_info'),
+        FieldPanel('parent'),
+        FieldPanel('related_subcategories',
+                   widget=forms.CheckboxSelectMultiple),
+    ]
+
+    def __str__(self):
+        return "{}: {}".format(self.parent.name, self.name)
+
+    class Meta:
+        ordering = ['weight']
+        verbose_name_plural = "Subcategories"
```

Comment:
Nitpicky: I'm not sure [this is supposed to be uppercase](https://docs.djangoproject.com/en/1.11/ref/models/options/#verbose-name-plural); I thought its use in the admin should uppercase as necessary.